play/src/front/Components/Input/InputTags.svelte

```diff
@@ -80,9 +80,9 @@
         --clear-select-color="red"
         --input-color="white"
         --chevron-icon-colour="white"
-        containerStyles="background-color:unset"
         inputStyles={"box-shadow:none !important;margin:0"}
         inputAttributes={{ "data-testid": testId }}
+        class="!bg-contrast !rounded-md !border-contrast-400 !outline-none !w-full"
```

Comment:
I'm not a huge fan of overriding the base theme this way.

Maybe we have a problem with the theme or the Select component?

I would like as much as possible to avoid putting Tailwind classes out of the base components. Could we discuss about the underlying issue?